emmaa/aws_lambda_functions/update_lambda.py

```diff
@@ -1,31 +1,40 @@
 import boto3
+import sys
 from os import path
 from zipfile import ZipFile
 
 HERE = path.dirname(path.abspath(__file__))
+script_name = sys.argv[1]
+function_name = sys.argv[2]
```

Comment:
I would suggest making a `main()` function where these values are extracted. Otherwise a linter or other IDE won't error if these names are used anywhere in this file, which could lead to major confusion down the road. Keeping these out of the global scope will prevent that.